amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -442,6 +452,12 @@ function exportScorecard () {
 										<digi:trn>No Data</digi:trn>
 									</c:if>
 								</td>
+									<td>
+										<c:if test="${not empty log.objectId && log.objectType=='org.digijava.module.aim.dbentity.AmpActivityVersion'}">
+											<input type="button" title="Click to compare with previous version" onclick="javascript:viewDifferences(${log.objectId})"
+												   class="dr-menu" value="&nbsp;&nbsp;<digi:trn>Compare</digi:trn>&nbsp;&nbsp;">

```

Comment:
Click to compare with previous version should be inside a digi:trn tag so its translated